common/cont-lib/usr/share/cont-docs/70-general.txt

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run THIS_IMAGE container-usage` to get this help.
+
+Run `docker run -ti THIS_IMAGE bash` to obtain interactive shell.
+
+Run `docker exec -ti CONTAINERID container-entrypoint` to access already running container.
```

Comment:
we still don't like the `container-entrypoint` helper as enabler for SCL collections.... 

@smarterclayton do you know if there is something we can do in `docker exec` to set the SCL vars before you enter container namespace?